app/build.gradle

```diff
@@ -27,7 +27,8 @@ dependencies {
     implementation 'com.jakewharton.rxbinding2:rxbinding-appcompat-v7:2.1.1'
     implementation 'com.jakewharton.rxbinding2:rxbinding-design:2.1.1'
     implementation 'com.facebook.fresco:fresco:1.10.0'
-    
+    implementation 'com.drewnoakes:metadata-extractor:2.11.0'
```

Comment:
Could you elaborate on the need for this library?

Reply:
@misaochan I have just updated the PR description to explain why I have added the library.